.github/CONTRIBUTING.md

```diff
@@ -51,7 +53,7 @@ To build and run a Concourse cluster from source, run the following in the root
 of this repo:
 
 ```sh
-$ docker-compose up
+docker-compose up
```

Comment:
Ok this is petty but I kinda like having the `$`s there. Sometimes it makes copy-pasting annoying but since these are all one-liners I figured it'd be a non-issue. (I guess it still borks triple-click.)

Reply:
I'll put 'em back! I originally took it out because my example for loading a pipeline was a single code block with three commands and I wanted it to be easily cut-pastable (and for everything to be consistent).

Reply:
No worries! Fixed.